pkg/operator/operator.go

```diff
@@ -55,6 +58,7 @@ func NewOpenShiftControllerManagerOperator(
 	c := &OpenShiftControllerManagerOperator{
 		targetImagePullSpec:  targetImagePullSpec,
 		operatorConfigClient: operatorConfigClient,
+		proxyLister:          proxyLister,
```

Comment:
don't you also need to add an event handler to the proxy informer below so you get notified on updates to the proxyconfig?

Reply:
ah, that appears to be handled in observeconfigcontroller